src/gradient.rs

```diff
@@ -141,6 +141,18 @@ pub struct Gradient {
     pub kind: GradientKind,
     /// Extend mode.
     pub extend: Extend,
+    /// The color space to be used for interpolation.
+    ///
+    /// The colors in the color stops will be converted to this color space.
+    ///
+    /// This defaults to [sRGB](ColorSpaceTag::Srgb).
```

Comment:
Defaults are a powerful thing. Is it plausible that OkLab would be a better default here?

Reply:
I don't think so. I checked with Raph on this earlier and he agreed this was probably the right default (for now?).

I'd also considered the potentially surprising behavior of "Are all ColorStops the same? If so and one hasn't been set explicitly, use that." but that is too much, I think.

But we can re-litigate this at any point in the future too.